openbas-front/src/admin/components/common/form/TextFieldAskAI.tsx

```diff
@@ -191,17 +176,24 @@ const TextFieldAskAI: FunctionComponent<TextFieldAskAiProps> = ({
   };
 
   const renderButton = () => {
+    const isAvailable = isEnterpriseEdition && enabled && configured;
     return (
       <>
-        <EETooltip forAi={true} title={t('Ask AI')}>
+        <EETooltip
+          forAi
+          title={isAvailable ? t('Ask AI') : t('Ask AI (EE)')}
```

Comment:
nitpick: We could spare a translation this way (the chinese translation is inconsistent as it is)
```suggestion
          title={t('Ask AI') + isAvailable ? '' : '(EE)'}
```